scheduler/reconcile_test.go

```diff
@@ -5490,14 +5535,15 @@ func TestReconciler_Disconnected_Client(t *testing.T) {
 			},
 		},
 		{
-			name:                         "stop-original-alloc-with-old-job-version-and-failed-replacements",
+			name:                         "stop-original-alloc-with-old-job-version-and-failed-replacements-replaced",
```

Comment:
I thought this test was redundant because, in the end, the replacement of the replacements were running, but it exercises the path where a reconnecting alloc has multiple replacements.